tests/templates/kuttl/kerberos/12-rego-rules.yaml.j2

```diff
@@ -0,0 +1,234 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: |
+      kubectl apply -n $NAMESPACE -f - <<EOF
```

Comment:
suggestion: if we call this via a file rather than via an inline kubectl apply it is easier to check indentation etc. - like it is done here: 
https://github.com/stackabletech/trino-operator/blob/main/tests/templates/kuttl/authentication/12-create-trino-cluster.yaml#L8

Reply:
substenv

Reply:
Implemented in efd0cec